bin/run_controller

```diff
@@ -150,11 +152,11 @@ if [ $IMAGE == "forch/controller" ] && [[ $faucet_id =~ ^[0-9]+$ ]]; then
     data_intf_name=data_faucet_$faucet_id
     sudo ip link add $data_intf_name type veth peer name data0 netns $pid
     sudo ip link set $data_intf_name up
-    docker exec $CONTAINER ip link add link data0 name data0.272 type vlan id 272
-    docker exec $CONTAINER ip addr add 192.168.2.$faucet_id dev data0.272
+    docker exec $CONTAINER ip addr add 192.168.2.$faucet_id dev data0
```

Comment:
Most if this "if image=controller" section should be moved into the docker file startup script, not as individual docker exec commands (this includes some things that were already here before this PR).

Reply:
I am leaving in the commands linked to data0